integration_tests/cork_test.go

```diff
@@ -135,9 +127,15 @@ func (s *IntegrationTestSuite) TestCork() {
 		queryClient := types.NewQueryClient(clientCtx)
 		res, err := queryClient.QueryCellarIDs(context.Background(), &types.QueryCellarIDsRequest{})
 		s.Require().NoError(err)
-		s.Require().Len(res.CellarIds, 1)
+		found := false
```

Comment:
Is there a reason we shouldn't know the exact expected length of `CellarIds`?

Reply:
It was another thing that was broken by a change to the genesis state, and I opted to remove it rather than increment the number since this test doesn't need to be aware of what cellars other tests need? It just seemed weird.  What do you think?

Reply:
Yeah I guess all that matters is that the one we need.